packages/p2p/src/pages/my-ads/my-ads-row-renderer.jsx

```diff
@@ -346,7 +346,7 @@ const MyAdsRowRenderer = observer(({ row: advert }) => {
                         <div onClick={onClickShare}>
                             <Popover
                                 alignment='bottom'
-                                className={classNames('p2p-my-ads__table-popovers__share', {
+                                className={classNames('my-ads-table__popovers__share', {
                                     'p2p-my-ads__table-popovers--disable':
```

Comment:
is this supposed to be `my-ads-table__popovers--disable`?

Reply:
oh yes it is 👍